packages/svelte/src/reactivity/map.js

```diff
@@ -28,7 +27,8 @@ export class ReactiveMap extends Map {
 			var sources = this.#sources;
 
 			for (var [key, v] of value) {
-				sources.set(key, source(v));
+				sources.set(key, source(Symbol()));
```

Comment:
Shouldn't we check if the `key` is an object here as well?

Reply:
By the way, I'm not sure if we need to fill the `sources` Map here, since we now generate signals in `has` and `get`

Reply:
Good point, fixed.